Core/GDCore/IDE/WholeProjectRefactorer.h

```diff
@@ -456,6 +456,26 @@ class GD_CORE_API WholeProjectRefactorer {
       const gd::EventsFunctionsExtension& eventsFunctionsExtension,
       const gd::EventsBasedObject& eventsBasedObject);
 
+  /**
+   * \brief Remove all the instances from one layer.
+   */
+  static void RemoveLayer(gd::Project &project, gd::Layout &layout,
+                          const gd::String &layerName);
+
+  /**
+   * \brief Move all the instances from one layer into another.
+   */
+  static void MergeLayers(gd::Project &project, gd::Layout &layout,
```

Comment:
The name of the method and the description do not mean the same things to me.
`MoveInstancesFromALayerToAnother`?

Reply:
The operation is merging 2 layers: "origin" layer and "target" layer into "target" layer.